lib/src/rules/always_declare_return_types.dart

```diff
@@ -63,21 +63,35 @@ class AlwaysDeclareReturnTypes extends LintRule implements NodeLintRule {
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
+  static const LintCode functionCode = const LintCode(
+      "always_declare_return_types", // ignore: prefer_single_quotes
```

Comment:
The convention in both analyzer and front_end is to always use double quotes for messages so that it's easier to use single quotes inside the message. Having to switch back and forth is a pain, but if we really want to enforce this lint, then I can update the existing strings to get rid of the 'ignore'.

Reply:
No worries.  I'm Ok with it as is.